GSA/openacr-editor
# 261 add chapter help text

Closes https://github.com/GSA/openacr/issues/261

Add yaml file with chapter introductory help text.
Create component to display help text and add that component to chapters and about page.
Add test to chapter suite to check that help text exists on those pages.

## QA
- Go to https://gsa.github.io/openacr-editor/
- Confirm that the correct chapter help text is displaying at the top of each page. [Mockup] (https://github.com/GSA/openacr/issues/261#issuecomment-1015720548) and [text](https://github.com/GSA/openacr/issues/261#issuecomment-1016449119) (note: minor text changes further in thread)
- Double check that text content is correct and is free of typos, including checking links in help text that includes links
- Confirm the new test in the chapter suite is 👍  and runs
